nixos/tests/plasma5.nix

```diff
@@ -17,7 +17,8 @@ import ./make-test.nix ({ pkgs, ...} :
         user = "alice";
       };
     };
-    services.xserver.desktopManager.kde5.enable = true;
+    services.xserver.desktopManager.plasma5.enable = true;
+    services.xserver.desktopManager.default = "plasma5";
```

Comment:
If they use the auto-login feature, yes. I don't know why it is suddenly necessary.